models/User.go

```diff
@@ -852,6 +869,9 @@ func PlaceBidOrder(userId uint32, bid *Bid) (uint32, error) {
 
 	l.Infof("Created Bid order. BidId: %d", bid.Id)
 
+	user.Cash = user.Cash - orderFee
+	l.Debugf("Deducted cash from user's account. New balance: %d", user.Cash)
```

Comment:
"Deducted **order fee** from user's account" : Better log statement, will help us if (god forbid) we need to debug this.